lib/Cluster.js

```diff
@@ -463,8 +492,20 @@ class Cluster extends EventEmitter {
       req.push(config);
     }
     if (req.length) {
-      debug(this.constructor.NAME, 'configure reporting', req);
-      const { reports } = await super.configureReporting({ reports: req });
+      // Check if command should get manufacturerSpecific flag
+      const manufacturerId = this._checkForManufacturerSpecificAttributes(
+        Object.keys(attributes).map(n => {
+          const attr = this.constructor.attributes[n];
+          return attr.id;
+        }),
+      );
+
+      debug(this.constructor.NAME, 'configure reporting', req, manufacturerId ? `manufacturer specific id ${manufacturerId}` : '');
+
+      const configureReportingOptions = { reports: req };
+      if (manufacturerId) configureReportingOptions.manufacturerId = manufacturerId;
```

Comment:
Is this check necessary here or could `configureReporting` do this check?
note: same at `readAttributesOptions`, `writeAttributeOptions` and `readReportingConfigurationOptions`

Reply:
Hm yes, good suggestion.